ocaml/sdk-gen/powershell/autogen/README_51.md

```diff
@@ -0,0 +1,164 @@
+
+The XenServer PowerShell Module includes a cmdlet for each API call,
+so API documentation and examples written for other languages will apply equally
+well to PowerShell. In particular, the SDK Guide and the Management API Guide
+are ideal for developers wishing to use this module.
+
+This module is free software. You can redistribute and modify it under the
+terms of the BSD 2-Clause license. See LICENSE.txt for details.
+
+## Reference
+
+For XenServer documentation see <https://docs.xenserver.com>
+
+The XenServer Management API Reference is available at
+<https://docs.xenserver.com/en-us/xenserver/8/developer/management-api>
+
+The XenServer Software Development Kit Guide is available at
+<https://docs.xenserver.com/en-us/xenserver/8/developer/sdk-guide>
+
+A number of examples to help you get started with the SDK is available at
+<https://github.com/xenserver/xenserver-samples>
+
+For community content, blogs, and downloads, visit
+<https://www.xenserver.com/blogs> and <https://www.citrix.com/community/>
+
+To network with other developers using XenServer visit
+<https://discussions.citrix.com/forum/101-hypervisor-formerly-xenserver/>
+
+## Prerequisites
+
+This library requires .NET Framework 4.5 or greater and PowerShell 5.1 or greater.
+
+## Dependencies
+
+The XenServer PowerShell Module is dependent upon the following libraries:
+
+- Newtonsoft JSON.NET by James Newton-King (see <https://www.newtonsoft.com/>).
+  JSON.NET is licensed under the MIT license.
+
+- XenServer.NET by Cloud Software Group, Inc.
+  XenServer.NET is a complete SDK for XenServer, exposing the XenServer
+  API as .NET classes. It is written in C#.
+
+## Folder Structure
+
+This archive contains the following folders that are relevant to PowerShell users:
+
+- `XenServerPowerShell\XenServerPSModule`: this is the XenServer PowerShell
+  Module
+- `XenServerPowerShell\src`: contains the C# source code for the XenServer
+  cmdlets shipped as a Visual Studio project.
+
+## Getting Started
+
+1. Extract the contents of this archive.
+
+    Note that some web browsers may mark the SDK ZIP file as "blocked" during
+    the download. To import the module successfully you will need to unblock the
+    archive before extracting its contents. To unblock the archive, right-click
+    on it and launch the Properties dialog. Click the Unblock button, then the
+    Apply or OK button.
+
+2. Navigate to the extracted XenServerPowerShell directory and copy the whole
+    folder XenServerPSModule into your PowerShell modules directory.
+
+    - On Windows this will normally be `$env:UserProfile\Documents\WindowsPowerShell\Modules`
+    for per-user configuration, or `$env:windir\system32\WindowsPowerShell\v1.0\Modules` for
+    system-wide configuration.
+
+    For more information see PowerShell's documentation on module paths:
+
+        PS> Get-Help about_PSModulePath
+
+3. Open a PowerShell 5 prompt as administrator.

```

Comment:
I think the version might not be needed for the old powershell, but if you want to use it, let's write 5.1 to be more precise. Same below.